### PR TITLE
Feat/manager 7392: anthos tracking

### PR DESCRIPTION
**Pull request by mohammed-zahaf**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/anthos`
| Bug fix?         | yes
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-7392
| License          | BSD 3-Clause

## Description
Anthos tracking

### CODE REVIEW

File: packages/manager/modules/anthos/src/dashboard/components/add-storage/component.js [hunk 1]
```diff
@@ -6,6 +6,9 @@ export default {
     goBack: '<',
     serviceName: '<',
     storageUsage: '<',
+    trackClick: '<',
+    trackPage: '<',
+    addStorageHitTracking: '<',
```
Comment (ganeshkumar1989) [inline packages/manager/modules/anthos/src/dashboard/components/add-storage/component.js:11]:
```suggestion
    addStorageHitTrackingId: '<',
```

File: packages/manager/modules/anthos/src/dashboard/components/order-host/component.js [hunk 2]
```diff
@@ -1,8 +1,14 @@
 import template from './template.html';
+import controller from './controller';
 
 export default {
   bindings: {
     goBack: '<',
+    trackingPrefix: '<',
+    trackClick: '<',
+    trackPage: '<',
+    orderHostHitTracking: '<',
```
Comment (ganeshkumar1989) [inline packages/manager/modules/anthos/src/dashboard/components/order-host/component.js:10]:
```suggestion
    orderHostHitTrackingId: '<',
```

File: packages/manager/modules/anthos/src/dashboard/components/order-public-ip/component.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import controller from './controller';
+import template from './template.html';
+
+export default {
+  bindings: {
+    goBack: '<',
+    trackingPrefix: '<',
+    trackClick: '<',
+    trackPage: '<',
+    orderPublicIpHitTracking: '<',
```
Comment (ganeshkumar1989) [inline packages/manager/modules/anthos/src/dashboard/components/order-public-ip/component.js:10]:
```suggestion
    orderPublicIpHitTrackingId: '<',
```

File: packages/manager/modules/anthos/src/dashboard/general-information/general-information.component.js [hunk 4]
```diff
@@ -14,9 +16,13 @@ export default {
     goBack: '<',
     goToTenant: '<',
     goToOrderHost: '<',
+    goToOrderStorage: '<',
     goToAssignPrivateIp: '<',
     goToOrderPublicIp: '<',
     goToRenameService: '<',
+    generalInfoHitTracking: '<',
```
Comment (ganeshkumar1989) [inline packages/manager/modules/anthos/src/dashboard/general-information/general-information.component.js:23]:
```suggestion
    generalInfoHitTrackingId: '<',
```

File: packages/manager/modules/anthos/src/dashboard/host/component.js [hunk 5]
```diff
@@ -8,6 +9,10 @@ export default {
     goToRestartHost: '<',
     goToSetStateHost: '<',
     serviceName: '<',
+    trackClick: '<',
+    trackPage: '<',
+    hostHitTracking: '<',
```
Comment (ganeshkumar1989) [inline packages/manager/modules/anthos/src/dashboard/host/component.js:14]:
```suggestion
    hostHitTrackingId: '<',
```

File: packages/manager/modules/anthos/src/dashboard/host/reinstall/component.js [hunk 6]
```diff
@@ -6,6 +6,9 @@ export default {
     serviceName: '<',
     goBack: '<',
     host: '<',
+    trackPage: '<',
+    trackClick: '<',
+    reinstallHostHitTracking: '<',
```
Comment (ganeshkumar1989) [inline packages/manager/modules/anthos/src/dashboard/host/reinstall/component.js:11]:
```suggestion
    reinstallHostHitTrackingId: '<',
```

File: packages/manager/modules/anthos/src/dashboard/host/remove/component.js [hunk 7]
```diff
@@ -6,6 +6,9 @@ export default {
     serviceName: '<',
     goBack: '<',
     hostService: '<',
+    trackPage: '<',
+    trackClick: '<',
+    removeHostHitTracking: '<',
```
Comment (ganeshkumar1989) [inline packages/manager/modules/anthos/src/dashboard/host/remove/component.js:11]:
```suggestion
    removeHostHitTrackingId: '<',
```

File: packages/manager/modules/anthos/src/dashboard/host/restart/component.js [hunk 8]
```diff
@@ -6,6 +6,9 @@ export default {
     serviceName: '<',
     goBack: '<',
     host: '<',
+    trackPage: '<',
+    trackClick: '<',
+    restartHostHitTracking: '<',
```
Comment (ganeshkumar1989) [inline packages/manager/modules/anthos/src/dashboard/host/restart/component.js:11]:
```suggestion
    restartHostHitTrackingId: '<',
```

File: packages/manager/modules/anthos/src/dashboard/host/set-state/component.js [hunk 9]
```diff
@@ -6,6 +6,9 @@ export default {
     serviceName: '<',
     goBack: '<',
     host: '<',
+    trackPage: '<',
+    trackClick: '<',
+    setStateHitTracking: '<',
```
Comment (ganeshkumar1989) [inline packages/manager/modules/anthos/src/dashboard/host/set-state/component.js:11]:
```suggestion
    setStateHitTrackingId: '<',
```

File: packages/manager/modules/anthos/src/dashboard/ips/component.js [hunk 10]
```diff
@@ -13,6 +13,9 @@ export default {
     goToOrderPublicIp: '<',
     goToAssignPrivateIp: '<',
     goToRemovePrivateIp: '<',
+    trackClick: '<',
+    trackPage: '<',
+    ipHitTracking: '<',
```
Comment (ganeshkumar1989) [inline packages/manager/modules/anthos/src/dashboard/ips/component.js:18]:
```suggestion
    ipHitTrackingId: '<',
```

File: packages/manager/modules/anthos/src/dashboard/ips/remove-private-ip/remove-private-ip.component.js [hunk 11]
```diff
@@ -7,6 +7,9 @@ export default {
     goBack: '<',
     privateIPs: '<',
     privateIp: '<',
+    trackClick: '<',
+    trackPage: '<',
+    removePrivateIpHitTracking: '<',
```
Comment (ganeshkumar1989) [inline packages/manager/modules/anthos/src/dashboard/ips/remove-private-ip/remove-private-ip.component.js:12]:
```suggestion
    removePrivateIpHitTrackingId: '<',
```

File: packages/manager/modules/anthos/src/dashboard/storage/component.js [hunk 12]
```diff
@@ -8,6 +8,9 @@ export default {
     serviceName: '<',
     storageUsage: '<',
     storageVolumes: '<',
+    trackClick: '<',
+    trackPage: '<',
+    storageHitTracking: '<',
```
Comment (ganeshkumar1989) [inline packages/manager/modules/anthos/src/dashboard/storage/component.js:13]:
```suggestion
    storageHitTrackingId: '<',
```

File: packages/manager/modules/anthos/src/dashboard/storage/remove/component.js [hunk 13]
```diff
@@ -6,6 +6,9 @@ export default {
     serviceName: '<',
     goBack: '<',
     storage: '<',
+    trackClick: '<',
+    trackPage: '<',
+    removeStorageHitTracking: '<',
```
Comment (ganeshkumar1989) [inline packages/manager/modules/anthos/src/dashboard/storage/remove/component.js:11]:
```suggestion
    removeStorageHitTrackingId: '<',
```

**Review by radireddy — APPROVED** [review]

File: packages/manager/modules/anthos/src/dashboard/components/add-storage/component.js (hunk 1 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/anthos/src/dashboard/components/add-storage/component.js:11]: Thanks for suggestion, i am not sure the `...Id` is relevant.
Id is something that can have more than one value.

So `addStorageHitTracking` is good, what do you think ?

**Review by ganeshkumar1989 — APPROVED** [review]

**Pull request merged**